frontend/src/main/components/Jobs/JobsTable.js

```diff
@@ -18,7 +16,38 @@ export default function JobsTable({ jobs }) {
       Header: "Status",
       accessor: "status",
     },
-    PlaintextColumn("Log", (cell) => cell.row.original.log),
+    {
+      Header: "Log",
+      accessor: "log",
+      Cell: ({ cell }) => {
+        const log = cell.row.original.log;
+        if (!log) {
+          return (
+            <div data-testid={`JobsTable-cell-row-${cell.row.index}-col-Log`}>
+              No logs available
+            </div>
+          );
+        }
+        const logLines = log.split("\n");
+        const truncatedLog = logLines.slice(0, 10).join("\n");
+        return (
+          <div data-testid={`JobsTable-cell-row-${cell.row.index}-col-Log`}>
+            {logLines.length > 10 ? (
+              <>
+                <pre>{truncatedLog}</pre>
```

Comment:
Try this instead, and see if you get a nicer looking result:
```suggestion
              <Plaintext text={truncatedLog} />,
```